syntax = "proto3";

package ipcscan.grpc;

message ScanRequest {}

message ScanResponse {
  string result = 1;
}

service IPCScanService {
  rpc Scan(ScanRequest) returns (ScanResponse);
}
